pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceProducer.java

```diff
@@ -619,7 +660,7 @@ private static void runProducer(int producerId,
                         }
                     }
                     rateLimiter.acquire();
-
+                    semaphore.acquire();
```

Comment:
the same as above

Reply:
You mean rateLimiter is the same as semaphore ?